### ISSUE TITLE
Taking advantage of rabbitmq 'Direct reply-to'

### ISSUE THREAD

**Issue by jfromaniello** [elasticio/amqp-rpc#23]
## tl;dr
RabbitMQ has a **pseudo-queue** that can be used as the reply queue called `amq.rabbitmq.reply-to`. Using this queue is faster than declaring a transient queue on every client and easier to manage than creating a long-lived queue.

---

I have been experimenting with this library and reading all the documentation of rabbitmq.

This library seems to follow the example of "Tutorial 6":

https://www.rabbitmq.com/getstarted.html

which is good but I discover this bit: https://www.rabbitmq.com/direct-reply-to.html

> Where does the client's queue come from? The client can declare a single-use queue for each request-response pair. But this is inefficient; even a transient unmirrored queue can be expensive to create and then delete (compared with the cost of sending a message). This is especially true in a cluster as all cluster nodes need to agree that the queue has been created, even if it is unmirrored.

> So alternatively the client can create a long-lived queue for its replies. But this can be fiddly to manage, especially if the client itself is not long-lived.

> The direct reply-to feature allows RPC clients to receive replies directly from their RPC server, without going through a reply queue. ("Directly" here still means going through the same connection and a RabbitMQ node; there is no direct network connection between RPC client and RPC server processes.) 

I've been trying this and it works well with few small changes. 

I am wondering if you are willing to accept a PR changing the default behavior, instead of declaring a queue when no `repliesQueue` is provided, using `amq.rabbitmq.reply-to` directly.

The only reason I can think for not doing this is keeping the library agnostic to rabbitmq, in that case I might contribute an example instead.
